firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionException.java

```diff
@@ -58,17 +60,6 @@
   @NonNull private final Status status;
   @Nullable private final AppDistributionRelease release;
 
```

Comment:
Removing these because we should always pass a message to the FirebaseAppDistributionException since its customer facing.